payments_redsys/__init__.py

```diff
@@ -78,6 +78,7 @@ def __init__(self, *args, **kwargs):
         self.terminal = kwargs.pop('terminal')
         self.shared_secret = kwargs.pop('shared_secret')
         self.currency = kwargs.pop('currency', '978')
+        self.ds_merchant_directpayment = kwargs.pop('ds_merchant_directpayment', 'TRUE').upper()

```

Comment:
Will be better if we name it `direct_payment`, `ds_merchant_` seems redundant :)